docs/conf.py

```diff
@@ -15,9 +15,10 @@
 """Sphinx documentation builder."""
 
 # -- General configuration ---------------------------------------------------
+import datetime
 
 project = "Qiskit"
-copyright = "2019, Qiskit Development Team"  # pylint: disable=redefined-builtin
+copyright = f"2016-{datetime.date.today().year}, Qiskit Development Team"  # pylint: disable=redefined-builtin
```

Comment:
I know the development started probably in 2016, but the first version was released in 2017. Do we count 2016 as the starting year of Qiskit? It's good to change the year here but I think the year shown in production is actually 2021 because the docs is built by the meta package not in Terra. Can you open another PR in meta package? https://github.com/Qiskit/qiskit/blob/d461015695742d1b12936dee2302e8acc2e8d271/docs/conf.py#L40

Reply:
moved to 2017.

Reply:
done in 1bbb182a1062ad9b1ecc3c342fe89c32ba4736e3